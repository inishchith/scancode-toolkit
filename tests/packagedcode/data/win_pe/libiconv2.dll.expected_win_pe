{
  "APIVersion": "", 
  "Assembly Version": "", 
  "BuildDate": "", 
  "BuildVersion": "", 
  "Comments": "", 
  "Company": "", 
  "CompanyName": "GNU <www.gnu.org>", 
  "FileDescription": "LibIconv: convert between character encodings", 
  "FileVersion": "1.9.2.1519", 
  "Full Version": "", 
  "InternalName": "libiconv2", 
  "LegalCopyright": "2004 Free Software Foundation <www.fsf.org>", 
  "LegalTrademarks": "GNU, LibIconv, libiconv2", 
  "LegalTrademarks1": "", 
  "LegalTrademarks2": "", 
  "License": "This program is free software; you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License;see www.gnu.org/copyleft/lesser.html.", 
  "OriginalFilename": "libiconv2.dll", 
  "ProductName": "LibIconv", 
  "ProductVersion": "1.9.2.1519", 
  "WWW": "", 
  "extra_data": {
    "LibToolFileVersion": "4:0:2", 
    "SpecialBuild": "GNU for Win32 <gnuwin32.sourceforge.net>"
  }
}