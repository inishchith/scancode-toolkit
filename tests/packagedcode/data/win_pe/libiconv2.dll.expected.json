{
  "APIVersion": null, 
  "Assembly Version": null, 
  "BuildDate": null, 
  "BuildVersion": null, 
  "Comments": null, 
  "Company": null, 
  "CompanyName": "GNU <www.gnu.org>", 
  "FileDescription": "LibIconv: convert between character encodings", 
  "FileVersion": "1.9.2.1519", 
  "Full Version": null, 
  "InternalName": "libiconv2", 
  "LegalCopyright": "2004 Free Software Foundation <www.fsf.org>", 
  "LegalTrademarks": "GNU, LibIconv, libiconv2", 
  "LegalTrademarks1": null, 
  "LegalTrademarks2": null, 
  "License": "This program is free software; you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License;see www.gnu.org/copyleft/lesser.html.", 
  "OriginalFilename": "libiconv2.dll", 
  "ProductName": "LibIconv", 
  "ProductVersion": "1.9.2.1519", 
  "WWW": null, 
  "extra_data": {
    "LibToolFileVersion": "4:0:2", 
    "SpecialBuild": "GNU for Win32 <gnuwin32.sourceforge.net>"
  }
}